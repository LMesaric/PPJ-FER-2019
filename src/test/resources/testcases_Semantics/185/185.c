int a[2]={2};
char b[4]="x\"\0\n\t\_";
int main(int a)
{
return a;
}