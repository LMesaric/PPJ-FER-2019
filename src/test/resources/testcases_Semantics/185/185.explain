semantic: invalid escaped string 'x\"\0\n\t\_' on byte offset 10
note: '"x\"\0\n\t\_"' on line 2
