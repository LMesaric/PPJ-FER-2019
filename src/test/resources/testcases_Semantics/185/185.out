<primarni_izraz> ::= NIZ_ZNAKOVA(2,"x\"\0\n\t\_")
